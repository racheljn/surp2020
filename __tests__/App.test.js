// import { shallow } from "enzyme";
// import React from "react";
// import renderer from "react-test-renderer";
//
// import Sunshine from "../components/App.js";
//
// describe("With Enzyme", () => {
//   it('Function shows "Hello, Sunshine!"', () => {
//     const app = shallow(<Sunshine />);
//
//     expect(app.find("div").text()).toEqual("Hello, Sunshine!");
//   });
// });
//
// describe("With Snapshot Testing", () => {
//   it('Sunshine shows "Hello, Sunshine!"', () => {
//     const component = renderer.create(<Sunshine />);
//     const tree = component.toJSON();
//     expect(tree).toMatchSnapshot();
//   });
// });
