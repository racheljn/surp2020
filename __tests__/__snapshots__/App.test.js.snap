// // Jest Snapshot v1, https://goo.gl/fbAQLP
//
// exports[`With Snapshot Testing Sunshine shows "Hello, Sunshine!" 1`] = `
// <div>
//   Hello, Sunshine!
// </div>
// `;
